frontend/public/style/_overrides.scss

```diff
@@ -213,53 +213,113 @@ tags-input .autocomplete .suggestion-item em {
 
 // PatternFly 4 nav overrides
 
-$pf-l-page__active-sidebar--BackgroundColor: rgba(255,255,255,.07);
+.pf-l-page__sidebar {
+  --pf-l-page__sidebar--BackgroundColor: #{$color-pf-black-900};
+  --pf-l-page__sidebar--PaddingBottom: 0;
+  --pf-l-page__sidebar--PaddingTop: 0;
 
-.pf-c-nav__link {
-  --pf-c-nav__list-link--Color: #{$color-pf-black-300};
-}
+  overflow-y: auto !important;
 
-.pf-c-nav__list {
-  .pf-c-nav__link {
-    &:active {
-      --pf-c-nav__list-link--active--Color: #{$color-pf-white};
-    }
-    &:focus {
-      --pf-c-nav__list-link--focus--Color: #{$color-pf-white};
+  .pf-c-nav {
+    // List link
+    --pf-c-nav__list-link--active--after--BackgroundColor: var(--pf-c-nav__list-link--hover--after--BackgroundColor);
+    --pf-c-nav__list-link--active--BackgroundColor: var(--pf-c-nav__list-link--hover--BackgroundColor);
+    --pf-c-nav__list-link--active--Color: var(--pf-c-nav__list-link--hover--Color);
+    --pf-c-nav__list-link--after--Height: 100%;
+    --pf-c-nav__list-link--after--Left: 0;
+    --pf-c-nav__list-link--after--md--Left: var(--pf-c-nav__list-link--after--Left);
+    --pf-c-nav__list-link--after--Width: 3px;
+    --pf-c-nav__list-link--Color: #{$color-pf-black-300};
+    --pf-c-nav__list-link--focus--after--BackgroundColor: var(--pf-c-nav__list-link--hover--after--BackgroundColor);
+    --pf-c-nav__list-link--focus--BackgroundColor: var(--pf-c-nav__list-link--hover--BackgroundColor);
+    --pf-c-nav__list-link--focus--Color: var(--pf-c-nav__list-link--hover--Color);
+    --pf-c-nav__list-link--hover--after--BackgroundColor: transparent;
+    --pf-c-nav__list-link--hover--BackgroundColor: #{$color-pf-black-700};
+    --pf-c-nav__list-link--hover--Color: #{$color-pf-white};
+    --pf-c-nav__list-link--m-current--after--BackgroundColor: #{$color-pf-blue-300};
+    --pf-c-nav__list-link--m-current--Color: var(--pf-c-nav__list-link--hover--Color);
+    --pf-c-nav__list-link--md--PaddingLeft: var(--pf-c-nav__list-link--PaddingLeft);
+    --pf-c-nav__list-link--PaddingBottom: var(--pf-c-nav__list-link--PaddingTop);
+    --pf-c-nav__list-link--PaddingLeft: 40px;
```

Comment:
This feels like too much padding to me because the nav items don't align to anything and have more horizontal space than other elements.  16px for mobile and 32px desktop puts the nav items in line with the hamburger above.   (note I don't think one value for both is appropriate given paddings reduce for mobile/increase for desktop).

Reply:
Nice catch.